src/KeyVault/KeyVault/help/Set-AzKeyVaultAccessPolicy.md

```diff
@@ -512,6 +512,8 @@ The acceptable values for this parameter:
 - Recover
 - Purge
 - Rotate
+- SetRotationPolicy
```

Comment:
The values are generated from https://github.com/Azure/azure-powershell/blob/f7955797a4f5106a909e78e5e0f95db0d1b27134/src/KeyVault/KeyVault/Commands/SetAzureKeyVaultAccessPolicy.cs#L277. Please update the code and re-generate the doc.

Reply:
Thanks for the suggestions! But after I rerun `Update-MarkdownHelpModule`, there is no markdown change for this command

![image](https://github.com/Azure/azure-powershell/assets/2101973/9f13760f-b839-4368-a4da-2d5208d1e965)

however, some other untouched commands get updates

Not sure if there is any breaking issue for the tool. Could you help to check?